packages/segmented-control/src/SegmentedControl.story.tsx

```diff
@@ -21,7 +25,7 @@ export default {
     size: {
       control: {
         type: 'radio',
-        options: Object.values(Size),
+        options: [...Object.values(Size), 'small'],
```

Comment:
`options: [...Object.values(Size), ...Object.values(DeprecatedSize)]`?

Reply:
ah left that in let me rm

Reply:
i was testing but we don't want that in storybook